data/mysql/schema.sql

```diff
@@ -969,9 +969,11 @@ CREATE TABLE `ezimagefile` (
   `contentobject_attribute_id` int(11) NOT NULL DEFAULT '0',
   `filepath` longtext NOT NULL,
   `id` int(11) NOT NULL AUTO_INCREMENT,
+  `version` int(11) NOT NULL DEFAULT '0',
   PRIMARY KEY (`id`),
   KEY `ezimagefile_coid` (`contentobject_attribute_id`),
-  KEY `ezimagefile_file` (`filepath` (191))
+  KEY `ezimagefile_file` (`filepath` (191)),
+  KEY `ezimagefile_version` (`version`)
```

Comment:
speaking out of my nose here, but I would imagine we always use the version col together with the attr_id when doing queries, never alone...

Reply:
probably, so then a combined index performs better is what you are saying? As opposed to DB having to use several indexes.

Reply:
Yes.
Explain-plan is your best friend of course, and different dbs can choose different strategies for execution, but generally speaking an index should cover as many as possible of the columns that are in the 'where' clause of the most often used queries. Cols within the index should be sorted, less selective ones 1st.

Reply:
Good catch, thanks! I'll double-check but at the first look, it makes sense to do a combined index here.

Reply:
Based on the GW code I've reviewed I'd also say that we need a composite index here.

Reply:
Added in: https://github.com/ezsystems/ezpublish-kernel/pull/2835/commits/37aa042eb127738e7210609333e4c283c870a800.